test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/InitializeTestProjectAttribute.cs

```diff
@@ -37,7 +37,7 @@ public override void Before(MethodInfo methodUnderTest)
             }
 
             MSBuildIntegrationTestBase.Project = ProjectDirectory.Create(_originalProjectName, _testProjectName, _baseDirectory, _additionalProjects, _language);
-            MSBuildIntegrationTestBase.TargetFramework = _originalProjectName.StartsWith("ClassLibrary") ? "netstandard2.0" : "netcoreapp2.2";
+            MSBuildIntegrationTestBase.TargetFramework = _originalProjectName.StartsWith("ClassLibrary") ? "netstandard2.0" : "netcoreapp3.0";
```

Comment:
#yolo

Reply:
Yeah, this is fine for now. We should log an issue for 3.0 RCL support and solve this then.